FooBar.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class FooBar
+{
+
+    private function foo($given_number): string

```

Comment:
Here and after - usage of "_" is not justified. You need to follow PSR-12 guidelines https://www.php-fig.org/psr/psr-12/